### PR TITLE
FLARE-136: Fixed the exception in list_jobs when study is None

### PR DESCRIPTION
**Pull request by nvidianz**

### CODE REVIEW

File: nvflare/private/fed/server/job_cmds.py [hunk 1]
```diff
@@ -193,8 +193,8 @@ def _send_summary_list(self, conn: Connection, jobs: List[Job]):
             table.add_row(
                 [
                     job.meta.get("job_id"),
-                    job.meta.get("name"),
-                    job.meta.get("study_name"),
+                    job.meta.get("name") or "",
+                    job.meta.get("study_name") or "",
```
Comment (IsaacYangSLA) [inline nvflare/private/fed/server/job_cmds.py:197]: Maybe job.meta.get("study_name", "") is better.

Comment (nvidianz) [reply nvflare/private/fed/server/job_cmds.py]: Definitely better. Updated.

**Review by IsaacYangSLA — APPROVED** [review]
LGTM.  A small side note that `not empty string` is True.

**Comment by nvidianz** [comment]
> LGTM. A small side note that `not empty string` is True.

Yeah. That's the desired behavior.

**Pull request merged**